resources/client/functions.ts

```diff
@@ -0,0 +1,213 @@
+import { Delay } from '../utils/fivem';
+
+let prop = 0;
+let isPhoneOpen = false;
+let propCreated = false;
+let phoneModel = 'prop_amb_phone'; // Refered to in newphoneProp function. Requires custom phone being streamed.
+
+/* * * * * * * * * * * * *
+ *
+ *  Prop Deletion/Creation handling
+ *
+ * * * * * * * * * * * * */
+
+const newPhoneProp = async () => {
+  removePhoneProp(); //deletes the already existing prop before creating another.
+  if (!propCreated) {
+    RequestModel(phoneModel);
+
+    while (!HasModelLoaded(phoneModel)) {
+      await Delay(1);
```

Comment:
Honestly might be a good idea to abstract this like we did with `loadDict` but its fine for now.

Reply:
Will make note of this to do later then